pilot/pkg/model/jwks_resolver.go

```diff
@@ -50,14 +50,14 @@ const (
 
 	// JwtPubKeyExpireDuration is the expire duration for JWT public key in the cache.
 	// After this duration expire, refresher job will fetch key for the cached item again.
-	JwtPubKeyExpireDuration = time.Hour
+	JwtPubKeyExpireDuration = time.Minute * 5
```

Comment:
Can you use an env variable to allow user to control ? And I would leave the default to 1 hour - that's the typical expiration of a JWT. Any reason to make it 5 ?